mapping-jobs/cprd/cprd-test.json

```diff
@@ -187,6 +187,26 @@
           "sourceRef": "file-system"
         }
       }
+    },
+    {
+      "name": "aurum-drug-mapping",
```

Comment:
@Okanmercan99 , We should also add this mapping to the cprd-with-original-structure.json job, since that's the main job we run on the real data, right?

Reply:
Done.